crypto/evp/p_enc.c

```diff
@@ -22,15 +22,19 @@ int EVP_PKEY_encrypt_old(unsigned char *ek, const unsigned char *key,
                          int key_len, EVP_PKEY *pubk)
 {
     int ret = 0;
+    DSA *dsa = NULL;
```

Comment:
RSA here

Reply:
Both fixed. It suprises me that DSA can work on my local compile.